contracts/nft-loan/src/execute.rs

```diff
@@ -44,10 +44,17 @@ pub fn deposit_collaterals(
     let borrower = info.sender;
 
     // ensure atleas one asset has been provided
+    // additional check: check if the person is really owner + if approved to be handled by contract
+    // need to check if the loan > approval because this could lead to problems
```

Comment:
can you expand on what `loan > approval` means? is it when the assets in the loan is greater than the assets that are approved in a loan?

Reply:
It was related to the above comment, I added more info so you can disregard this one